lib/rubocop/cop/rails/content_tag.rb

```diff
@@ -3,30 +3,29 @@
 module RuboCop
   module Cop
     module Rails
-      # This cop checks that `tag` is used instead of `content_tag`
-      # because `content_tag` is legacy syntax.
+      # This cop checks legacy syntax usage of `tag`
       #
-      # NOTE: Allow `content_tag` when the first argument is a variable because
-      #      `content_tag(name)` is simpler rather than `tag.public_send(name)`.
+      # NOTE: Allow `tag` when the first argument is a variable because
+      #      `tag(name)` is simpler rather than `tag.public_send(name)`.
```

Comment:
Can you add the following mention?
```suggestion
      #      `tag(name)` is simpler rather than `tag.public_send(name)`.
      #      And this cop will be renamed to something like `LegacyTag` in the future. (e.g. RuboCop Rails 2.0)
```